backend/src/main/java/model/Order.java

```diff
@@ -0,0 +1,12 @@
+package model;
+
+
+import java.time.Instant;
+
+public class Order {
+
+    private int id;
+    private int customerID;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "customerID" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)